edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html

```diff
@@ -0,0 +1,84 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Would you like to take {{ display_name }} as a proctored exam?
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you're enrolled in this course as a verified student, you have the option to take this exam with
+      online proctoring. Online proctoring is one requirement towards being eligible for credit.
+    {% endblocktrans %}
+  </p>
+  <div class="gated-sequence">
+    <span><i class="fa fa-lock"></i></span>
+    <a class="start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
+      {% trans "Yes, I want to take this exam with online proctoring (and be eligible for credit)" %}
+    </a>
+    <p>
+      {% blocktrans %}
+        You will be guided through installing {{platform_name}} approved online proctoring software and
+        performing various checks to set up your proctored exam session. Have your photo ID
+        ready for the photo ID verification step.<br />
+        Immediately after you complete the set up, you will begin your timed and proctored exam.<br />
+        Before you set up your proctoring session, you might want to <a href="#">read this</a>
+        to learn more about taking a proctored exam.<br />
+      {% endblocktrans %}
+    </p>
+
+    <i class="fa fa-arrow-circle-right start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false></i>
+  </div>
+  <div class="gated-sequence">
+    <span><i class="fa fa-unlock"></i></span>
+    <a class="start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=false data-start-immediately=true>
+      {% trans "No, I want to take this exam without proctoring (and not be eligible for credit)" %}
+    </a>
+    <p>
+      {% blocktrans %}
+        By not taking the exam as a proctored exam, you will not earn credit for completing
```

Comment:
"If you take this exam without proctoring, you will not be eligible to purchase academic credit for this course, even if you achieve a passing grade."